packages/atomic/src/templates/default.html

```diff
@@ -17,16 +17,18 @@
   .content-container {
     width: 100%;
     display: grid;
+    grid-template-columns: minmax(0, 1fr);
```

Comment:
Small trick I learned: Without this, the results would overflow the page, since `display: grid` without any template for columsn mean: 1 column, auto size.

And auto size == "fit content".

With this, since there's now a default definite minimum value of 0... it magically adapt itself to the screen size instead.